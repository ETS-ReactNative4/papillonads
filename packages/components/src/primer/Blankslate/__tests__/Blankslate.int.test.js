import { renderDefault, renderNarrow, renderCapped, renderSpacious, renderLarge, renderCleanBackground } from './Blankslate.int.render'

describe('<Blankslate />', () => {
  describe('Render', () => {
    test('must match renderDefault()', () => {
      expect(global.renderToJSON(renderDefault())).toMatchSnapshot()
    })

    test('must match renderNarrow()', () => {
      expect(global.renderToJSON(renderNarrow())).toMatchSnapshot()
    })

    test('must match renderCapped()', () => {
      expect(global.renderToJSON(renderCapped())).toMatchSnapshot()
    })

    test('must match renderSpacious()', () => {
      expect(global.renderToJSON(renderSpacious())).toMatchSnapshot()
    })

    test('must match renderLarge()', () => {
      expect(global.renderToJSON(renderLarge())).toMatchSnapshot()
    })

    test('must match renderCleanBackground()', () => {
      expect(global.renderToJSON(renderCleanBackground())).toMatchSnapshot()
    })
  })
})
