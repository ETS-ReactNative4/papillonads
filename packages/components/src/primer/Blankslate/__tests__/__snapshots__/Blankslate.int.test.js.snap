// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Blankslate /> Render must match renderCapped() 1`] = `
<div
  className="blankslate blankslate-capped"
>
  <h3>
    This is a blank slate
  </h3>
  <p>
    Use it to provide information when no dynamic content exists.
  </p>
</div>
`;

exports[`<Blankslate /> Render must match renderCleanBackground() 1`] = `
<div
  className="blankslate blankslate-clean-background"
>
  <h3>
    This is a blank slate
  </h3>
  <p>
    Use it to provide information when no dynamic content exists.
  </p>
</div>
`;

exports[`<Blankslate /> Render must match renderDefault() 1`] = `
<div
  className="blankslate"
>
  <h3>
    This is a blank slate
  </h3>
  <p>
    Use it to provide information when no dynamic content exists.
  </p>
</div>
`;

exports[`<Blankslate /> Render must match renderLarge() 1`] = `
<div
  className="blankslate blankslate-large"
>
  <h3>
    This is a blank slate
  </h3>
  <p>
    Use it to provide information when no dynamic content exists.
  </p>
</div>
`;

exports[`<Blankslate /> Render must match renderNarrow() 1`] = `
<div
  className="blankslate blankslate-narrow"
>
  <h3>
    This is a blank slate
  </h3>
  <p>
    Use it to provide information when no dynamic content exists.
  </p>
</div>
`;

exports[`<Blankslate /> Render must match renderSpacious() 1`] = `
<div
  className="blankslate blankslate-spacious"
>
  <h3>
    This is a blank slate
  </h3>
  <p>
    Use it to provide information when no dynamic content exists.
  </p>
</div>
`;
